components/sensor/atm90e32.rst

```diff
@@ -0,0 +1,139 @@
+ATM90E32 Power Sensor
+====================
+
+.. seo::
+    :description: Instructions for setting up ATM90E32 energy metering sensors
+    :image: atm90e32.png
+    :keywords: ATM90E32, CircuitSetup, Split Single Phase Real Time Whole House Energy Meter, Expandable 6 Channel ESP32 Energy Meter Main Board
+
+The ``atm90e32`` sensor platform allows you to use your ATM90E32 voltage/current and power sensors
+(`datasheet <http://ww1.microchip.com/downloads/en/devicedoc/Atmel-46003-SE-M90E32AS-Datasheet.pdf>`__) sensors with
+ESPHome. This sensor is commonly found in CircuitSetup 2 and 6 channel energy meters.
+
+The atm90e32 IC can measure up to three AC voltages although typically only one
+voltage measurement would be used for the mains electricity phase of a
+household. Three current measurements are read via CT clamps. 
+
+The CircuitSetup Split Single Phase Real Time Whole House Energy Meter can read 2 current channels and one voltage channel.
+.. figure:: images/atm90e32-cs-2chan-full.jpg
+    :align: center
+    :width: 50.0%
+
+    CircuitSetup Split Single Phase Real Time Whole House Energy Meter.
+
+The CircuitSetup Expandable 6 Channel ESP32 Energy Meter can read 6 current channels and 2 voltage channels at a time, this board has two atm90e32 ICs and requires two sensors to be configured in ESPHome.
+.. figure:: images/atm90e32-cs-6chan-full.jpg
+    :align: center
+    :width: 50.0%
+
+    CircuitSetup Expandable 6 Channel ESP32 Energy Meter Main Board.
+
+The IC supports a number of measurements, these are not yet exported by ESPHome
```

Comment:
Let's not add what the FW can't do 😉  - I think there's not much against exposing these as sensors... Can we do that? Plus it looks like some of the things listed here (for example frequency) *are* supported by the integration already.